nats/nats_test.go

```diff
@@ -162,6 +162,25 @@ func Test_Storage_Nats_Set(t *testing.T) {
 	require.Len(t, keys, 1)
 }
 
+func Test_Storage_Nats_SetWithContext(t *testing.T) {
+	var (
+		testStore = New(config)
```

Comment:
bug: The new test functions are not using the testcontainers approach for creating a testStore.

Could you update this line (across all files) to follow each storage's pattern to create the testStore? See https://github.com/gofiber/storage/pull/1569/files#diff-34e731bdbdd3141a4cf0239cb4c1aa2f100ffd9563ad07e455ca439c551a0cf2R153 as reference.

We tried to be consistent in that, but some storages needed some minor differences in the pattern.